unlock-app/src/stories/creator/DemoPage.stories.js

```diff
@@ -4,28 +4,37 @@ import { storiesOf } from '@storybook/react'
 import Demo from '../../pages/demo'
 import createUnlockStore from '../../createUnlockStore'
 
+const myLock = {
+  address: '0xaaaaaaaaa0c4d48d1bdad5dcb26153fc8780f83e',
+  name: 'My Blog',
+  keyPrice: '27000000000000000',
+  expirationDuration: '172800',
+  maxNumberOfKeys: 240,
+  outstandingKeys: 3,
+}
 const store = createUnlockStore({
-  locks: {
-    '0x56781234a': {
-      address: '0x56781234a',
+  locks: [
+    {
+      address: '0xab7c74abc0c4d48d1bdad5dcb26153fc8780f83e',
       keyPrice: '10000000000000000000',
       expirationDuration: '86400',
       maxNumberOfKeys: '800',
       outstandingKeys: '32',
     },
-    '0x12345678a': {
-      address: '0x12345678a',
-      name: 'My Blog',
-      keyPrice: '27000000000000000',
-      expirationDuration: '172800',
-      maxNumberOfKeys: '240',
-      outstandingKeys: '3',
+    myLock,
+  ],
+  router: {
+    location: {
+      pathname: '/demo/' + myLock.address,
     },
   },
+  currency: {
+    USD: 195.99,
+  },
 })
 
 storiesOf('Demo', Demo)
   .addDecorator(getStory => <Provider store={store}>{getStory()}</Provider>)
   .add('the demo', () => {
-    return <Demo lockAddress="5678a" />
+    return <Demo lock={myLock} />
```

Comment:
So, the `Demo` component exported from `demo.js` does take a `lockAddress`, not the `lock` object.
If you look closely you'll see that there is indeed a `Demo` object but we actually export a `connect`-ed version of it which takes the lock address...

Reply:
again, same story, I did not change any source, just the stories to make them work, so perhaps some further investigation to figure out what is going on?

Reply:
Well you changed the passed argument from `lockAddress` to `lock` ;)

Reply:
Right, because that is actually what is required by prop types of Demo. I'm new to this part of the codebase, I'd love a bit of context as to what you are expecting/how this should work